library/objective-c/EnvoyEngine.m

```diff
@@ -51,6 +51,16 @@ - (EnvoyHTTPStream *)startStreamWithObserver:(EnvoyObserver *)observer {
 
 @end
 
+#pragma mark - EnvoyConfiguration
+
+@implementation EnvoyConfiguration
+
++ (NSString *)templateString {
+  return [[NSString alloc] initWithUTF8String:config_template];
```

Comment:
nit: how about putting
```
static NSString * const kEnvoyTemplateString = [[NSString alloc] initWithUTF8String:config_template];
```
above the `@implementation` and then just returning that each time.

Reply:
Unfortunately this doesn't work since we hit the following error:

> Initializer element is not a compile-time constant

Reply:
Ah, makes sense. Silly, since of course it is constant at compile-time, but I understand the limitations at this scope.

You _could_ work around this by using `#define` instead of the extern'd const... but, let's not worry about it.